charts/bpa/templates/_helpers.tpl

```diff
@@ -272,12 +272,18 @@ Create environment variables for database configuration.
 Return JAVA_OPTS -Dmicronaut.config.files
 */}}
 {{- define "bpa.config.files" -}}
-{{- if and .Values.keycloak.enabled .Values.schemas.enabled -}}
+{{- if and .Values.keycloak.enabled .Values.schemas.enabled .Values.ux.enabled -}}
```

Comment:
Maybe you could create a list `$configfiles` and add the filenames based on the enablement
`$configfiles | join ","`

Reply:
see also https://helm.sh/docs/chart_template_guide/function_list/#lists-and-list-functions

Reply:
(I think there is also some cases missing)

Reply:
that sounds good, I will take a look.

Reply:
Thanks for the pointer. I have changed it to use lists and it is much easier to read and understand.

Reply:
nice!